examples/keras_imagenet_resnet50.py

```diff
@@ -29,20 +56,21 @@
 config.gpu_options.visible_device_list = str(hvd.local_rank())
 K.set_session(tf.Session(config=config))
 
-# Settings from https://arxiv.org/abs/1706.02677.
-batch_size = 32
-learning_rate = 0.0125
-warmup_epochs = 5
-weight_decay = 0.00005
-epochs = 90
+# Default settings from https://arxiv.org/abs/1706.02677.
+train_batch_size = args.batch_size
```

Comment:
Let's inline these, and move the comment about default settings to where args are defined.

Reply:
Done.